nameserver/http.go

```diff
@@ -65,6 +67,26 @@ func ServeHTTP(listener net.Listener, version string, server *DNSServer, domain
 					return
 				} // oh, I already know this. whatever.
 			}
+
+			if dockerCli != nil {
+				running := true
+
+				// check with Docker whether the container is really running, removing otherwise...
+				Debug.Printf("[http] Checking whether '%s' is running...", idStr)
+				if container, err := dockerCli.InspectContainer(idStr); err != nil {
+					if _, notThere := err.(*docker.NoSuchContainer); notThere {
+						running = false
+					}
+				} else {
+					running = container.State.Running
```

Comment:
Interesting question. Whatever we do here should be consistent with our processing of the event stream. I don't think we pay attention to 'pause' transitions there, do we? Perhaps we should. raise an issue.